tabular/src/autogluon/tabular/models/tab_transformer/hyperparameters/searchspaces.py

```diff
@@ -0,0 +1,10 @@
+from autogluon.core import Real
+
+# TODO: I don't see a need to split search space by problem type.
+#  Is there a reason to at this point?
+def get_default_searchspace():
+    params = {
+        'lr': Real(5e-5, 5e-3, log=True)
```

Comment:
set default of each HP in the search-space to match the default HP value used without HPO. In this case: `default = 1e-3`

Reply:
Got it. Added defaults to each continuous hyperparameter.